src/components/CourseSideBar.vue

```diff
@@ -47,7 +47,7 @@ const navs = [
     </li>
   </ul>
   <div v-else class="scrollbar-hide w-full overflow-scroll">
-    <div class="tabs w-max">
+    <div class="tabs m-auto w-max">
```

Comment:
why is it `m-auto` but not `mx-auto`?

Reply:
updated. I just didn't think much about it